### DISCUSSION TITLE
Crop service with specifying the file name

### DISCUSSION THREAD

**Discussion by francescocerasuolo** [developmentseed/titiler#391] [Feature Request]
I'm developing a visual widget on my web application to select a crop area on the map and download the cropped image of a product in geoTIFF.

So, I dynamically create urls like this:
/stac/crop/88.027,22.365,88.714,22.9482/100x92.tif?url=[...]/TEST-calibrated.json&assets=overview-vea&rescale=0%2C249&resampling_method=average

I noticed that the file will be downloaded on the browser always with the name **[width]x[height].[format]**. In the example it will be downloaded a file with name "100x92.tif".

Is there a way to customize the file name to the server?

**Comment by kylebarron** [comment]
Browsers tend to automatically save files where the name is everything after the last `/` and before the query string (`?`). IMO titiler shouldn't need to change to support this. If you're downloading a file from the command line, use `wget url -O filename.tif`. If you're downloading the file in a browser, use "Save File As"
